PULL_REQUEST_TEMPLATE.MD

```diff
@@ -3,6 +3,6 @@
 Thanks for contributing to the project. Please follow the guidelines
 below:
 
-* Make sure you've read the [Contributing guide](CONTRIBUTING.md).
+* Make sure you've read the [Contributing guide](https://github.com/UdacityMobileWebScholarship/blood-donation/blob/dev/CONTRIBUTING.md).
```

Comment:
CONTRIBUTING.md  should be from master branch.

Reply:
@optimistanoop any specific reason for this?

Reply:
@varunzxzx because master our main branch. What is your view on this?

Reply:
@optimistanoop The contribution guide is same in both the branches.

Reply:
@varunzxzx I think it's just a matter of choice, I preferred to go with single source (master).

Reply:
Yeah, but I think we should target working branch. Do it anyway you like :v:

Reply:
but this may need work at the time deleting dev (if needed, or else we will have to keep dev to support this), also it will have dev branch link once it will merge to master. Ideally, we should have dev link in dev branch and master branch link in master branch so that a branch need not to depend on other.